gotechnica/platform-2020
# Already Submitted Project

### Description
1. Create a new project submission item called "project_submitted" that is added to the DB for a team when the team is created. When the "Submit My Project" button is clicked in the projects page, this item should be updated as "true". 
2. Hackers should not be able to click the "Submit My Project" button until all checkbox items in the list on the page are checked. The button should be greyed out and un-clickable until this condition is fulfilled.
3. When a hacker loads the project submission page and the "project_submitted" item is true for their team, the page should notify them their project has already been submitted per the mocks. Specifically, the first screen's button text should be updated to read " Project has already been submitted" and it should not be clickable.

created get_team_submission and update_team_submission endpoints 

addressed ticket: https://trello.com/c/xPIDSZtv/320-projects-as-a-hacker-i-want-to-know-if-other-members-of-my-team-have-already-submitted-our-project

### Steps to Test

1. Go to my teams tab and create a team
2. Go to Submit Project tab and press "I'm Ready to Submit" button
3. Click on any of the project submission inputs/checkboxes and see if it will allow you to submit (should include both checkboxes and text inputs)
4. After submitting, try to go to submit project tab again and see that the button will be disabled and will say that the project has already been submitted

Endpoint: 
GET - https://bqjydw3dzk.execute-api.us-east-1.amazonaws.com/dev/get_team_submission?team_id=af99b5cb-a0ff-411b-accf-6c2c872b52e9

POST - https://bqjydw3dzk.execute-api.us-east-1.amazonaws.com/dev/update_team_submission
body:
`{
        "team_id": "af99b5cb-a0ff-411b-accf-6c2c872b52e9",
        "project_submitted": false
}`
